.travis.yml

```diff
@@ -14,7 +14,7 @@ addons:
       - g++-4.8
 
 before_script:
-  - npm install -g gulp
-  - for d in `ls packages/`; do cd packages/$d; npm install; cd -; done
+  - npm install -g gulp lerna
+  - lerna bootstrap
```

Comment:
YES. I actually meant to leave this comment on one of the PRs from yesterday. `bootstrap` seems like the way it should be.